src/test/java/com/wikia/webdriver/elements/mercury/pages/ErrorPage.java

```diff
@@ -0,0 +1,44 @@
+package com.wikia.webdriver.elements.mercury.pages;
+
+import com.wikia.webdriver.common.logging.PageObjectLogging;
+import com.wikia.webdriver.elements.common.Navigate;
+import com.wikia.webdriver.elements.mercury.components.Navigation;
+import com.wikia.webdriver.pageobjectsfactory.pageobject.WikiBasePageObject;
+
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.FindBy;
+
+public class ErrorPage extends WikiBasePageObject {
```

Comment:
Please access the components present on the Error Page by lombok as described in the architecture doc:
https://wikia-inc.atlassian.net/wiki/display/AUT/How+to+build+a+Page+from+Components+using+Lombok